src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/FishboneMoncriefDisk.hpp

```diff
@@ -317,6 +317,10 @@ class FishboneMoncriefDisk {
     return equation_of_state_;
   }
 
+  const gr::Solutions::KerrSchild& background_spacetime() const noexcept {
```

Comment:
it's not need now, but it'll be used for retrieving the `background_spacetime` for the magnetized disk. Since the later is just a FM disk + a B-field, I find retrieving the background spacetime from `FishboneMoncriefDisk` more consistent than hard-coding the same background spacetime in the magnetized FM class. If for some reason this sounds unnecessary or stupid, I can leave this commit out.

Reply:
I don't see why it's ever needed. You can always grab the spacetime solution from the `variables` function. We already needed this for the evolution. This function was explicitly removed from all the analytic solutions so that all variables are treated on equal footing.

Reply:
commit was removed